build/passes-windows.ps1

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+$ErrorActionPreference = 'Continue'
+
+# Installing requirements for CI
+cd ../src/Passes/
+pip install -r requirements.txt
+
+
+
+# Installing Clang, CMake and LLVM
+if (!(Get-Command clang -ErrorAction SilentlyContinue)) {
+    & (Join-Path $env:CONDA scripts conda.exe) install -c conda-forge llvm-tools=11.1.0
+    & (Join-Path $env:CONDA scripts conda.exe) install -c conda-forge llvmdev=11.1.0
+}
+
+refreshenv
+$env:Path += ";C:\Miniconda\Library\LLVM\bin\"
+$env:LLVM = "C:\Miniconda\Library\LLVM\"
+
+cmd.exe /c "call `"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat`" && set > %temp%\vcvars.txt"
```

Comment:
Given that this script is WiP, I am inclined to solve this offline, mostly because I have no way of making changes and testing that they are OK. If it is OK with you, I will move this script to another branch and I would be happy to have your help to finish it once I have access to a Windows machine. Does that work for you?